File: app/src/main/java/com/beemdevelopment/aegis/database/AppDatabase.java
package com.beemdevelopment.aegis.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.beemdevelopment.aegis.data.db.MyCreationDao;
import com.beemdevelopment.aegis.model.MyGalleryModel;

@Database(entities = {AuditLogEntry.class, MyGalleryModel.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract AuditLogDao auditLogDao();
    public abstract MyCreationDao myCreationDao();
}

File: app/src/main/java/com/kuemiin/reversevoice/data/db/AppDatabase.kt
package com.kuemiin.reversevoice.data.db

import androidx.room.Database
import androidx.room.RoomDatabase
import com.kuemiin.reversevoice.model.MyGalleryModel

@Database(
    entities = [MyGalleryModel::class],
    version = 1,
    exportSchema = false
)
abstract class AppDatabase : RoomDatabase() {
    abstract fun myCreationDao(): MyCreationDao
}
